doc/manuals/orion-api.md

```diff
@@ -1038,6 +1039,68 @@ Note that the `attrs` and `metadata` fields can be used also in subscriptions (a
 with the same meaning to specify which attributes (metadata) to include in notifications associated
 to that subscription.
 
+## Oneshot Subscription
+
+Oneshot subscription provides an option to subscribe an entity only for one time notification. When consumer creates a subscription 
+with status `oneshot`, a subscription is created as similar to the [normal subscription](user/walkthrough_apiv2.md#subscriptions) request with a slight difference.
+
+In the normal case, the consumer gets initial and continuous notifications whenever the entity is updated until subscription is removed or its status passes to inactive after a subscription update.
+
+While, in the case of oneshot subscription, the consumer gets notified only one time whenever the entity is updated after creating 
+the subscription. Once a notification is triggered, the subscription transitions to `status`: `inactive`. Once in this status, 
+the consumer may update it with `oneshot`∫ to repeat the same behavior (i.e. to get the one time notification again). 
+
+![](user/oneshot_subscription.png "oneshot_subscription.png")
```

Comment:
Do I need to move this image? If yes, where?

Reply:
Tipically images should be in the same place where the .md using them is (although they are some exceptions, i.e. a png used from two .md files which doesn't belong to the same directory).

In this case, it should be moved to the same place where orion-api.md is.

Reply:
Done in 23ac2483c613079a9c96c2245f2a00faea4c8a9f